src/main/resources/api/service_openapi.yaml

```diff
@@ -702,6 +712,14 @@ components:
       items:
         $ref: '#/components/schemas/RoleBinding'
 
+    GrantRoleRequestBody:
+      description: The body for a request to grant a role to a single user.
+      type: object
+      required: [memberEmail]
```

Comment:
thinking out loud: you could make a batch endpoint that takes a list of emails. It would look very similar to this. That can come later.

Reply:
we actually [talked about writing a batch api](https://docs.google.com/document/d/1mC2AhU3Rw8zvOTYtxdLyfQtcrIHjQjXBgvmMBD-I7As/edit?usp=sharing) that looks pretty similar to this one, though it raised some interesting questions around transactions and how they're supported in Sam